zqd/handlers_zeek_test.go

```diff
@@ -38,9 +39,11 @@ func TestPcapPostSuccess(t *testing.T) {
 	if runtime.GOOS == "windows" {
 		t.Skip("skipping test for windows")
 	}
+	pcapuri, err := iosrc.ParseURI("./testdata/valid.pcap")
```

Comment:
```suggestion
	pcapuri, err := iosrc.ParseURI("testdata/valid.pcap")
```

Reply:
Out of curiosity, why is this better? Isn't `./` more explicit?

Reply:
To be clear, this is fine as is, and you should keep it if you prefer it. My take is just that a ./ prefix is redundant in this context.